### DISCUSSION TITLE
how to change the buffering indicator texture?

### DISCUSSION THREAD

**Discussion by junguler** [tomasklaen/uosc#709] [Q&A] [answered]
hello, i want to change the texture and maybe the size as well, it would awesome to know where should i look, thanks

**Comment by tomasklaen** [comment, marked as answer]
The fork button I guess.

If you're asking for the code, it's in `elements/BufferingIndicator.lua`. Changing the size should be simple, but shape not so much.

**Reply by junguler** [reply]
thanks for your reply, i guess changing size might be enough too
